controller/environment_blackbox_test.go

```diff
@@ -65,6 +79,14 @@ func (s *EnvironmentControllerSuite) TestCreate() {
 		require.NotNil(t, env)
 		assert.Equal(t, env.Data.ID, newEnv.Data.ID)
 	})
+
+	s.T().Run("cluster_not_linked", func(t *testing.T) {
+		spaceID := uuid.NewV4()
+		payload := newCreateEnvironmentPayload("osio-stage", "stage", "cluster2.com")
+
+		_, err := test.CreateEnvironmentInternalServerError(t, s.ctx, s.svc, s.ctrl, spaceID, payload)
```

Comment:
see question above about the InternalServerError returned when the cluster is not linked to the user.